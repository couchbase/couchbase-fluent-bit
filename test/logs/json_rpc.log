[json_rpc:debug,2021-03-09T17:32:02.093Z,ns_1@cb.local:json_rpc_connection-goxdcr-cbauth<0.678.0>:json_rpc_connection:handle_call:158]sending jsonrpc call:{[{jsonrpc,<<"2.0">>},
                       {id,0},
                       {method,<<"AuthCacheSvc.UpdateDB">>},
                       {params,
                        [{[{specialUser,<<"@goxdcr-cbauth">>},
                           {nodes,
                            [{[{host,<<"127.0.0.1">>},
                               {user,<<"@goxdcr">>},
                               {password,"*****"},
                               {ports,
                                [8091,18091,11210,11207,8092,18092,9999,9999]},
                               {local,true}]}]},
                           {tlsConfig,{[{present,false}]}},
                           {authCheckURL,<<"http://127.0.0.1:8091/_cbauth">>},
                           {permissionCheckURL,
                            <<"http://127.0.0.1:8091/_cbauth/checkPermission">>},
                           {permissionsVersion,
                            <<"vFYFjkwj1H8dcOqQxpXXEBrMwxE=">>},
                           {authVersion,<<"VVGFbtVaKOqBqbJFrklZL8MR6V0=">>},
                           {certVersion,2575464363},
                           {extractUserFromCertURL,
                            <<"http://127.0.0.1:8091/_cbauth/extractUserFromCert">>},
                           {clientCertAuthState,<<"disable">>},
                           {clientCertAuthVersion,
                            <<"cAN8XSFAtQ98DdNbAKta8iJSRpw=">>},
                           {clusterEncryptionConfig,
                            {[{encryptData,false},
                              {disableNonSSLPorts,false}]}}]}]}]}
[json_rpc:debug,2021-03-09T17:32:02.094Z,ns_1@cb.local:json_rpc_connection-goxdcr-cbauth<0.678.0>:json_rpc_connection:handle_info:94]got response: [{<<"id">>,0},{<<"result">>,true},{<<"error">>,null}]
[json_rpc:debug,2021-03-09T17:32:15.459Z,ns_1@cb-example-0000.cb-example.default.svc:json_rpc_connection-fts-service_api<0.1310.0>:json_rpc_connection:handle_call:158]sending jsonrpc call:{[{jsonrpc,<<"2.0">>},
                       {id,2},
                       {method,<<"ServiceAPI.GetTaskList">>},
                       {params,[{[{rev,<<"MA==">>},{timeout,30000}]}]}]}
[json_rpc:debug,2021-03-09T17:32:15.463Z,ns_1@cb-example-0000.cb-example.default.svc:json_rpc_connection-fts-service_api<0.1310.0>:json_rpc_connection:handle_info:94]got response: [{<<"id">>,3},
               {<<"result">>,
                {[{<<"rev">>,<<"Mw==">>},
                  {<<"nodes">>,[<<"6fc08c5818ffa52dc6c624eda31bddc1">>]},
                  {<<"isBalanced">>,true}]}},
               {<<"error">>,null}]
