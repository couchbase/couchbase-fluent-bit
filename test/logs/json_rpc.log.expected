/fluent-bit/test/logs/json_rpc.log.actual: [1615311122.093000000, {"file_path":"/fluent-bit/test/logs","file":"json_rpc.log","logger":"json_rpc","timestamp":"2021-03-09T17:32:02.093Z","message":"ns_1@cb.local:json_rpc_connection-goxdcr-cbauth<0.678.0>:json_rpc_connection:handle_call:158]sending jsonrpc call:{[{jsonrpc,<<\"2.0\">>},\n                       {id,0},\n                       {method,<<\"AuthCacheSvc.UpdateDB\">>},\n                       {params,\n                        [{[{specialUser,<<\"@goxdcr-cbauth\">>},\n                           {nodes,\n                            [{[{host,<<\"127.0.0.1\">>},\n                               {user,<<\"@goxdcr\">>},\n                               {password,\"*****\"},\n                               {ports,\n                                [8091,18091,11210,11207,8092,18092,9999,9999]},\n                               {local,true}]}]},\n                           {tlsConfig,{[{present,false}]}},\n                           {authCheckURL,<<\"http://127.0.0.1:8091/_cbauth\">>},\n                           {permissionCheckURL,\n                            <<\"http://127.0.0.1:8091/_cbauth/checkPermission\">>},\n                           {permissionsVersion,\n                            <<\"vFYFjkwj1H8dcOqQxpXXEBrMwxE=\">>},\n                           {authVersion,<<\"VVGFbtVaKOqBqbJFrklZL8MR6V0=\">>},\n                           {certVersion,2575464363},\n                           {extractUserFromCertURL,\n                            <<\"http://127.0.0.1:8091/_cbauth/extractUserFromCert\">>},\n                           {clientCertAuthState,<<\"disable\">>},\n                           {clientCertAuthVersion,\n                            <<\"cAN8XSFAtQ98DdNbAKta8iJSRpw=\">>},\n                           {clusterEncryptionConfig,\n                            {[{encryptData,false},\n                              {disableNonSSLPorts,false}]}}]}]}]}","hostname":"test-container","couchbase":{"logging.version":"99","operator.version":"unknown","server.version":"unknown","node-config":"unknown","server.server":"unknown","analytics":"false","cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","index":"false","query":"false","search":"false","eventing":"false","data":"false"},"logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","filename":"/fluent-bit/test/logs/json_rpc.log","pod":{"namespace":"default","name":"test-pod","uid":"unknown"},"level":"DEBUG","level_numeric":"10"}]
/fluent-bit/test/logs/json_rpc.log.actual: [1615311122.094000000, {"file_path":"/fluent-bit/test/logs","file":"json_rpc.log","logger":"json_rpc","timestamp":"2021-03-09T17:32:02.094Z","message":"ns_1@cb.local:json_rpc_connection-goxdcr-cbauth<0.678.0>:json_rpc_connection:handle_info:94]got response: [{<<\"id\">>,0},{<<\"result\">>,true},{<<\"error\">>,null}]","hostname":"test-container","couchbase":{"logging.version":"99","operator.version":"unknown","server.version":"unknown","node-config":"unknown","server.server":"unknown","analytics":"false","cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","index":"false","query":"false","search":"false","eventing":"false","data":"false"},"logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","filename":"/fluent-bit/test/logs/json_rpc.log","pod":{"namespace":"default","name":"test-pod","uid":"unknown"},"level":"DEBUG","level_numeric":"10"}]
/fluent-bit/test/logs/json_rpc.log.actual: [1615311135.459000000, {"file_path":"/fluent-bit/test/logs","file":"json_rpc.log","logger":"json_rpc","timestamp":"2021-03-09T17:32:15.459Z","message":"ns_1@cb-example-0000.cb-example.default.svc:json_rpc_connection-fts-service_api<0.1310.0>:json_rpc_connection:handle_call:158]sending jsonrpc call:{[{jsonrpc,<<\"2.0\">>},\n                       {id,2},\n                       {method,<<\"ServiceAPI.GetTaskList\">>},\n                       {params,[{[{rev,<<\"MA==\">>},{timeout,30000}]}]}]}","hostname":"test-container","couchbase":{"logging.version":"99","operator.version":"unknown","server.version":"unknown","node-config":"unknown","server.server":"unknown","analytics":"false","cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","index":"false","query":"false","search":"false","eventing":"false","data":"false"},"logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","filename":"/fluent-bit/test/logs/json_rpc.log","pod":{"namespace":"default","name":"test-pod","uid":"unknown"},"level":"DEBUG","level_numeric":"10"}]
/fluent-bit/test/logs/json_rpc.log.actual: [1615311135.463000000, {"file_path":"/fluent-bit/test/logs","file":"json_rpc.log","logger":"json_rpc","timestamp":"2021-03-09T17:32:15.463Z","message":"ns_1@cb-example-0000.cb-example.default.svc:json_rpc_connection-fts-service_api<0.1310.0>:json_rpc_connection:handle_info:94]got response: [{<<\"id\">>,3},\n               {<<\"result\">>,\n                {[{<<\"rev\">>,<<\"Mw==\">>},\n                  {<<\"nodes\">>,[<<\"6fc08c5818ffa52dc6c624eda31bddc1\">>]},\n                  {<<\"isBalanced\">>,true}]}},\n               {<<\"error\">>,null}]","hostname":"test-container","couchbase":{"logging.version":"99","operator.version":"unknown","server.version":"unknown","node-config":"unknown","server.server":"unknown","analytics":"false","cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","index":"false","query":"false","search":"false","eventing":"false","data":"false"},"logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","filename":"/fluent-bit/test/logs/json_rpc.log","pod":{"namespace":"default","name":"test-pod","uid":"unknown"},"level":"DEBUG","level_numeric":"10"}]
